spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -237,6 +237,8 @@ object Engine {
   // For `context` in ThreadPool, it is the called thread when poolSize is 1.
   // So many usages of that thread, we will not change it for now.
   val dnnComputing: ThreadPool = new ThreadPool(1)
+  // We need to init dnn thread in case that users directly call model operation in java local
+  initDnnThread()
```

Comment:
I think we should push the =init= at the beginning or the end. It will be much clearer.